### PR TITLE
added resident_key option in settings

### PR DESCRIPTION
**Pull request by variable**
This is to implement feature request https://github.com/CZ-NIC/django-fido/issues/138

### CODE REVIEW

File: django_fido/settings.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class DjangoFidoSettings(AppSettings):
         certificate=NestedListSetting(inner_setting=FileSetting(), default=[]),
         crl_list=NestedListSetting(inner_setting=FileSetting(), default=[]),
     ), default=None)
+    resident_key = BooleanSetting(default=False)
```
Comment (tpazderka) [inline django_fido/settings.py:38]: This should also be noted in `CHANGELOG` and maybe explained in `README`.

**Review by tpazderka — CHANGES_REQUESTED** [review]
Quality is also failing...

File: django_fido/settings.py (hunk 1 above)
Comment (variable) [reply django_fido/settings.py:38]: I actually don't know what the cause of the error, the unittests went fine

Comment (variable) [reply django_fido/settings.py:38]: I thought I'd leave it changelog untouched until a release is ready, I can add some stuff in README

Comment (tpazderka) [reply django_fido/settings.py:38]: Run `tox -equality`, there is an extra line in one of the files.

We have an `Unreleased` section in `CHANGELOG` for exactly this.

Comment (variable) [reply django_fido/settings.py:38]: Alright, I will have a look tomorrow

Comment (variable) [reply django_fido/settings.py:38]: Fixed quality check, updated changelog and readme

**Comment by tpazderka** [comment]
It looks like you need to rebase to current master to fix conflicts, then I can run the CI again.

**Comment by variable** [comment]
> It looks like you need to rebase to current master to fix conflicts, then I can run the CI again.

done

**Review by tpazderka — CHANGES_REQUESTED** [review]
Please remove the `.idea` folder from the commit. Otherwise it looks good.

**Comment by variable** [comment]
> Please remove the `.idea` folder from the commit. Otherwise it looks good.

removed.

File: .gitignore [hunk 2]
```diff
@@ -107,3 +107,6 @@ node_modules/
 
 # Ignore transpiled JS code
 django_fido/static/django_fido/js
+
```
Comment (tpazderka) [inline .gitignore:110]: This should be in your user `.gitignore` and not in project one. This should remain language specific.

**Comment by variable** [comment]
Removed from .gitignore

File: .gitignore (hunk 2 above)
Comment (variable) [reply .gitignore]: Removed .idea from .gitignore

**Review by tpazderka — APPROVED** [review]

**Comment by tpazderka** [comment]
Merged manually.

**Pull request closed without merging**